src/Entities/Transaction.php

```diff
@@ -139,6 +147,54 @@ public function getPaymentCardId()
     }
 
     /**
+     * @return array
+     */
+    public function getChildTransactions()
+    {
+        return $this->getAttribute('childTransactions');
+    }
+
+    /**
+     * @return array
+     */
+    public function getInvoiceIds()
+    {
+        return $this->getAttribute('invoiceIds');
+    }
+
+    /**
+     * @return array
+     */
+    public function getSubscriptionIds()
+    {
+        return $this->getAttribute('subscriptionIds');
+    }
+
+    /**
+     * @return PaymentMethodInstrument
+     */
+    public function getPaymentInstrument()
+    {
+        return $this->getAttribute('paymentInstrument');
```

Comment:
This won't return instance of `PaymentMethodInstrument`, you need to add factory like [this one](https://github.com/Rebilly/rebilly-php/blob/master/src/Entities/Payment.php#L181)